specification/communications-manager.yaml

```diff
@@ -377,6 +377,8 @@ paths:
             * `*/*`
             * `application/json`
             * `application/vnd.api+json`
+
+            Where no `Accept` header is present, this will default to `application/vnd.api+json`
```

Comment:
I think we should expand on the documentation further up in the API description where we mention the acceptable types the server can generate, maybe adding in:

* A list of valid `Accept` header values
* Information on which will be default when no `Accept` header is present

Reply:
Thanks @seanmarjason-nhs !